src/utils/validation/registration/registrationValidation.ts

```diff
@@ -45,7 +45,8 @@ export const registrationValidationSchema = Yup.object<YupShape<Registration>>({
     npiSubjectHeading: Yup.string()
       .nullable()
       .when('$publicationInstanceType', ([publicationInstanceType], schema) =>
-        isBook(publicationInstanceType) || isChapter(publicationInstanceType)
+        nviApplicableTypes.includes(publicationInstanceType) &&
+        (isBook(publicationInstanceType) || isChapter(publicationInstanceType))
```

Comment:
ikke testet så skal ikke si for sikkert, men blir ikke dette det samme?
```suggestion
        nviApplicableTypes.includes(publicationInstanceType)
```

Reply:
Ja, ligger to ekstra inni nviApplicableTypes. Tenker også denne løsningen er grei mtp evt. endringer i framtiden på hva som er NVI-kategorier (om det noen gang endres da).